### ISSUE TITLE
Is there a way to use ImGui's Freetype support?

### ISSUE THREAD

**Issue by ctrlcctrlv** [imgui-rs/imgui-rs#359]

**Comment by dbr** [comment]
As of imgui-rs 0.7 there isn't an easy way to enable [imgui's freetype support](https://github.com/ocornut/imgui/tree/master/misc/freetype)

With imgui 1.81 (imgui 0.7 uses 1.80), the freetype integration was simplified. Once imgui-rs is updated to use >=1.81, then this might be fairly easy to do..

What I tried:

1. Add define for `IMGUI_ENABLE_FREETYPE` into `imgui-sys/build.rs`
2. Include `imgui_freetype.cpp` into `imgui-sys/src/include_all_imgui.cpp` (this also required `build.include("third-party/imgui");` so `imgui_freetype.cpp` could find `imgui.h`)
3. Add in the appropriate compiler/linker flags for freetype (I'm not sure the best cross-platform approach to this is - maybe good idea to look at the `-sys` crate for `freetype-rs` or `rust-freetype`?)
4. Add a feature flag to opt into this (there's already an example in `build.rs` to respect a feature flag in build.rs)

Then in theory that is about it, I think.. however I did all that got some compiler errors from the C++ part along the lines of
```
.../imgui-rs/imgui-sys/./third-party/cimgui/imgui/misc/freetype/imgui_freetype.cpp:147: undefined reference to `FT_New_Memory_Face'
```
..which I suspect may be to do with the order everything is getting passed to the compiler, although I didn't look into it closely
